content/60.methods.md

```diff
@@ -54,87 +105,35 @@ For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %
-
-##### Assessment
-
-###### Outcome Assessment
-
-Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
-They described whether there were any missing data such as whether there were individuals lost to follow up.
-They then describe whether there were any potential sources of bias such as lack of blinding in a randomized controlled trial.
-
-###### Statistical Methods Assessment
-
-Reviewers described which statistical methods were used for inference and whether applied methods were appropriate for the study.
-They then described whether adjustments were made for possible confounders.
+<!-- To Do: Describe Ryan's work making it possible to cite clinical trial identifiers-->
+<!-- To Do: Also, if we end up automating the Manubot-to-LaTeX workflow while trying to submit this paper, we should describe that too-->
+Due to the needs of this project, project contributors also implemented new features in Manubot and [Zotero](https://www.zotero.org/), which Manubot uses to extract metadata for some types of citations.
+These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899] and generating the complete review manuscript along with the individual manuscripts reviewing specific topics. <!-- To Do: reference individual manuscripts -->
 
-##### Results
+<!--To Do: This is old text, separate out into intro what was pre-existing and here what was added over the course of this project-->
+Finally, Manubot and GitHub Actions continuous integration allowed for scripted updates to be run each time the manuscript was generated.
+These scripts were used to check that the manuscript was built correctly, run spellchecking, and cross-reference the manuscripts cited in this review, summarized in Appendix A, and discussed in the project's issues and pull requests.
 
-Reviewers described the estimated association between the treatment and outcome.
-They described measures of confidence or statistical significance, if provided.
+### Results
 
-##### Interpretation
+#### Recruitment and Community Building
 
-Reviewers described whether a causal claim could be made.
-They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.
+<!--To Do: Add a graph/text describing number of unique contributors & commits over time-->
+<!-- To Do: Word Cloud of #17 to show diversity of interests?-->
+<!--To Do: plot number of article issues opened over time-->
 
-##### Extrapolation
+#### Data Analysis and Integration
 
-Reviewers describe how the study may extrapolate to a different species or population.
+<!--To Do: I guess the result here would be just to provide the code that generates the figures? Depending on mSystems licensing, perhaps we can provide an example?-->
 
 ### Conclusions
```

Comment:
@mprobson I haven't updated these, just moved the existing text. Conclusions will depend on what we do above! However, I did drop some blockchain links for you 😆

Reply:
Do we know why the first one was retracted? I don't think the second one is actually blockchain based but rather focused on repeatability/reliability in code (and input data).

Reply:
It was retracted because Daniel Himmelstein showed they could not actually prove their protocol existed as originally claimed.  The [manuscript comments](https://f1000research.com/articles/5-222#article-comments) summarize this, and there is more detail in Daniel's [blog](https://blog.dhimmel.com/irreproducible-timestamps/).

I'll give it more thought, but getting into blockchain for data provenance feels out of scope for this particular manuscript.

Reply:
Sorry @agitter, I was partially joking because @mprobson's students are all presenting about blockchain this semester, and he's sick of talking about it. I just thought it was funny that people are worried about blockchain for data provenance in medicine when there is so, so much else we could be doing for reproducibility (like making code for figures available, i.e., manubot!) If it is included, it would just be a throwaway comment -- those articles are the primary type of thing that came up when I was searching for open publishing in medicine, and it just seems like a really weird emphasis for the literature to be taking based on all the work that could be done!

Reply:
Ha, got it.  Ironically, Manubot actually does use blockchain for provenance via OpenTimestamps
> Manubot supports OpenTimestamps to timestamp the HTML and PDF outputs on the Bitcoin blockchain. This procedure allows one to retrospectively prove that a manuscript version existed prior to its blockchain-verifiable timestamp

(from https://greenelab.github.io/meta-review/#timestamping)